### PR TITLE
Remove Highly

### PR DESCRIPTION
**Pull request by fbernhart**
highly

**Issue:** #3169 
**Alexa rank:** -

### ~Checklist~
  ~- [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)~
  ~- [x] I optimized the icon with SVGO or SVGOMG~
  ~- [x] The SVG `viewbox` is `0 0 24 24`~

### ~Description~
~I took the already used SVG.~
~I used Inkscape to fix size and center~
~I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG~

### Edit
This PR is removing the Highly icon as the brand no longer exists. ---> https://twitter.com/highlytm

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
It appears Highly has closed down - could you look at removing this ready for the next major release? 🖊

**Comment by fbernhart** [comment]
@adamrusted Done ⚰️

**Comment by PeterShaggyNoble** [comment]
@fbernhart, could you update the original post to clarify that this PR will remove the icon, please?

File: .svglint-ignored.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
...
     "M1.74219 17.9588C0.78125 17.9588 0 18.7528 0 19.7294C0 20.6981 0.78125 21.5 1.74219 21.5C2.70312 21.5 3.48438 20.6981 3.48438 19.7294C3.48438 18.7528 2.71094 17.9588 1.74219 17.9588ZM23.1797 2.5L12.2188 6.05704V2.54764L0.390624 6.38257V16.5694L10.5938 13.2585V16.752L24 12.4089L20.3672 8.80422L23.1797 2.5ZM10.5938 11.5117L2.02344 14.2906V7.58943L10.5938 4.81049V11.5117ZM20.9688 11.6546L12.2188 14.4971V7.8038L20.2734 5.18366L18.4219 9.15357L20.9688 11.6546Z": "Clubhouse",
-    "M10.2 20.4H13.799999999999999V24H10.2zM9 0H15V18H9z": "Highly",
+    "M 16.046875 5.039062 L 24 12.992188 L 21.292969 15.703125 L 16.046875 10.457031 L 14.203125 12.308594 L 11.488281 9.597656 Z M 10.652344 10.4375 L 13.359375 13.152344 L 15.90625 15.691406 L 13.191406 18.398438 L 7.953125 13.160156 L 3.777344 17.335938 L 2.707031 18.398438 L 0 15.691406 L 7.953125 7.738281 Z M 10.652344 10.4375": "Code Climate",
```
Comment (PeterShaggyNoble) [inline .svglint-ignored.json:52]: Was this addition intentional?

Comment (fbernhart) [reply .svglint-ignored.json:52]: I'm not sure how any why this addition ended up here. 😄  Strange.

But let me remove it.

**Review by PeterShaggyNoble — APPROVED** [review]
Looks good to me, thanks, @fbernhart 👍🏻
